### PR TITLE
Add io.github.mak448a.QTCord

### PR DESCRIPTION
**Pull request by mak448a**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by mak448a** [comment]
By the way, I can't get the icon to work. Could any of you please help?

**Comment by hfiguiere** [comment]
submission are to be made against the `new-pr` branch

File: flatpak-pip-generator
Comment (hfiguiere) [inline flatpak-pip-generator:1]: this doesn't belong here.

File: io.github.mak448a.QTCord.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      - install -Dm 644 smiley.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - ls ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/
+      #- ls /app/share/icons/hicolor/128x128/apps
+      - mv QTCord.desktop /app/share/applications/io.github.mak448a.QTCord.desktop
+      - mv * /app/bin
+      - ls /app/bin
+    sources:
+      # All files from here will be moved to the final flatpak
+      - type: dir
+        path: includes
```
Comment (hfiguiere) [inline io.github.mak448a.QTCord.yml:26]: upstream source doesn't belong here.

**Comment by mak448a** [comment]
Oh ok sorry

**Comment by mak448a** [comment]
I'll fix it

**Comment by mak448a** [comment]
I'm going to reopen this on new-pr

**Pull request closed without merging**